UIKitTasks/UIKitTasks/ProfileScreen/Cells/HeaderTableViewCell.swift

```diff
@@ -0,0 +1,226 @@
+// HeaderTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Нужен для того, чтобы открыть браузер
+protocol Openable: AnyObject {
+    /// Открытие браузера
+    func openBrowser()
+}
+
+/// Ячейка основного профиль
+final class HeaderTableViewCell: UITableViewCell {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let myProfileText = "Мария Иванова"
+        static let profileDesctiptionText = "Консультат"
+        static let linkText = "📎www.spacex.com"
+        static let plusSymbol = "+"
+        static let editText = "Изменить"
+        static let shareText = "Поделиться профилем"
+        static let cornerRadiusAddButton = 13.0
+    }
+
+    // MARK: - Visual Components
+
+    private let myProfileImageView = {
```

Comment:
```suggestion
    private let profileImageView = {
```
вроде нельзя так писать типа myProfile или myImage
Просто мой профиль не совсем понятно, возможно нужно дописать типа профиль страницы приложения, но это не точно